src/wp-includes/blocks/legacy-widget.php

```diff
@@ -24,13 +24,9 @@ function render_block_core_legacy_widget( $attributes ) {
 		return '';
 	}
 
-	if ( method_exists( $wp_widget_factory, 'get_widget_object' ) ) {
-		$widget_object = $wp_widget_factory->get_widget_object( $attributes['idBase'] );
-	} else {
-		$widget_object = gutenberg_get_widget_object( $attributes['idBase'] );
-	}
 
-	if ( ! $widget_object ) {
+	$widget_key = $wp_widget_factory->get_widget_key( $attributes['idBase'] );
```

Comment:
Note: `get_widget_key()` does not exist in Core yet. It's a suggested change in [PR 1378](https://github.com/WordPress/wordpress-develop/pull/1378/files#diff-14ab2708608a0b1542b959b470b83d8fb3f442fdf1bb15eaf2b8d6034d5620f5R131) for Trac ticket 53427.